gnomad_qc/v3/sample_qc/v2_pc_relate.py

```diff
@@ -0,0 +1,58 @@
+from gnomad_qc.v2.resources import qc_mt_path, get_gnomad_meta
+from gnomad_qc.v3.resources import gnomad_v2_qc_sites, v3_qc, meta, v2_v3_pc_relate_pca_scores, v2_v3_relatedness
+import hail as hl
+import argparse
+import logging
+
+logger = logging.getLogger("v2_pc_relate")
+
+
+def liftover_v2_qc_mt() -> hl.MatrixTable:
```

Comment:
is this the only code that requires the lifted over v2 qc mt? just wondering why this function is in this code and not in a more general location

Reply:
hmmmm, that's a good point! How about moving this to `gnomad_qc.v2.resources.sample_qc.py` ?